### ISSUE TITLE
Bug: Cannot move cards on Chrome

### ISSUE THREAD

**Issue by mgdelacroix** [mattermost/focalboard#1936]
## Steps to reproduce the behavior

1. On any board, try to move a card to a different column
2. Observe how the card doesn't move, and no XHR request is fired on the Network tab

## Expected behavior

The card should move normally

## Screenshots

If applicable, add screenshots to elaborate on the problem.

## Edition and Platform

 - Edition: Mattermost plugin
 - Version: v0.11.0 RC1
 - Browser and OS: Chrome 96.0.4664.45 on Linux

## Additional context

Other updates to cards like title changes work perfectly, so this points to a drag & drop issue

**Comment by chenilim** [comment]
Followup from triage: We weren't able to repro this issue on Chrome (Linux and Mac). @agnivade, are there any specific errors you see in the browser console or network tab?

**Comment by agnivade** [comment]
Nope. You can see more debugging info here: https://community-daily.mattermost.com/core/pl/q864yzdyotbn5cdap7uocrwe8e.

No console errors. No websocket events. No XHR. Absolutely nothing. I can post a gif tomorrow.

**Comment by agnivade** [comment]
Tried again today, and it's working again. This has been going on for a while. I'll post a video whenver I repro this. But it definitely happens for sure.

**Comment by agnivade** [comment]
Funnily, happened again today, but when I tried to capture a video, it started working. There's some non-deterministic part here.

**Comment by agnivade** [comment]
Ok reproed it. Gif attached:
![repro](https://user-images.githubusercontent.com/1774000/145754142-9b2ae2d9-fd20-4e88-b32c-6ad2c792d2ab.gif)

**Comment by sbishel** [comment]
HI @agnivade Are you still seeing this issue. We have not been able to duplicate..thanks.

**Comment by agnivade** [comment]
Yes, just tried it and it became even worse. It goes to some other Chrome tab.

**Comment by chenilim** [comment]
Thanks Agniva! What do you mean by "it goes to some other Chrome tab"? Does another tab open?

Also, what Linux distro / version is running? As best we can tell, this may be specific to Chrome on that OS.

This also seems to be specific to drag-and-drop. The other place we use it is rearranging the card's content blocks (e.g. description). If you have a moment, does this also repro:
1. Create a card
2. Set a description, e.g. "Text 1"
3. Hover below and select "Add content", select Text
4. Enter "Text 2"
5. Drag and drop Text 2 above Text 1

https://user-images.githubusercontent.com/46905241/149232517-783ed814-cf65-4bf1-a128-ff7e6369ddb2.mov

Does that work?

**Comment by agnivade** [comment]
> What do you mean by "it goes to some other Chrome tab"? Does another tab open?

Yeah, when I try to drop the card, it moves to another Chrome tab and tries to drop it as a file.

I am running Ubuntu 20.04.

> Does that work?

Nope, that also doesn't work. Yes, it's clearly a drag-drop issue. I cannot do any drag-drop movement in focalboard.

**Comment by chenilim** [comment]
Thanks @agnivade! Putting this up for grabs. cc @jespino - there may be a set of Ubuntu / Linux specific issues.
